src/Symfony/Component/DomCrawler/Tests/AbstractCrawlerTest.php

```diff
@@ -354,6 +354,18 @@ public function testText()
         $this->assertSame('my value', $this->createTestCrawler(null)->filterXPath('//ol')->text('my value'));
     }
 
+    /**
+     * Tests that innerText() returns only text that is the direct descendent of the current node, in contrast to
+     * text() that returns the text of all child nodes.
+     */
```

Comment:
The comment is not needed

Reply:
A test scenario is not needed, in a technical sense, but it is needed in a maintainability sense. It conveys what the original intent of the test is, which helps greatly in determining how to fix it when it should break in future.

Reply:
From my POV we can keep it, but `testInnerText()` method name makes clear what's the point here 🤷‍♂️

Reply:
Perhaps, but I think writing test scenarios is a good habit to be into.

Reply:
Do we use this anywhere else in the DomCrawler's test suite? If not, I am in favour of removing the comment and, if necessary, change the method name to something like `testInnerTextReturnsContentOfDirectDescendantTextNodesExclusively()`.

Reply:
Strongly disagree. Prose is much easier to read as space-separated words instead of stuffing entire sentences into camel-cased method names.

Reply:
in this case `testInnerText()` is enough